runbotics/runbotics-orchestrator-ui/src/src-app/components/ProcessVariablesPanel/PositionedSnackbar.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```

Comment:
Hm, is this a new snackbar component? We already have one, to see usage search in project enqueueSnackbar